DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/IntitialMenuViewModelTests.cs

```diff
@@ -148,7 +148,7 @@ public void Initial_menu_should_have_list_of_section_card_view_model()
             var expectedCourseContent = CourseContentHelper.CreateDefaultCourseContent();
             var section = CourseSectionHelper.CreateDefaultCourseSection(sectionName: sectionName, hasLearning: hasLearning, percentComplete: percentComplete);
             expectedCourseContent.Sections.Add(section);
-            var expectedSection = new SectionCardViewModel(section);
+            var expectedSection = new SectionCardViewModel(section, 1);

```

Comment:
The customisation id should be set when creating the course content and then that variable referenced again here. Otherwise it's not clear what this 1 represents or where it's coming from 🙂 Same for all the other tests